packages/kit/src/exports/public.d.ts

```diff
@@ -1012,6 +1012,10 @@ export interface RequestEvent<
 	 * related to the data request in this case. Use this property instead if the distinction is important to you.
 	 */
 	isDataRequest: boolean;
+	/**
+	 * `true` if the request comes from a call to one of our own API endpoints made during SSR (or prerendering). This is handled by directly calling `+server.js` without the overhead of actually making an HTTP request. Use this property if you want to treat these synthetic requests differently in some way.
```

Comment:
I might drop the last sentence as it doesn't really say anything
```suggestion
	 * `true` if the request comes from a call to one of our own API endpoints made during SSR (or prerendering). SvelteKit in this case directly calls `+server.js` without the overhead of actually making an HTTP request.
```

Reply:
Yeah, I like this better. Done in 4124d78648b29b770c92a5ecf12457ea8aa88960.